server/backend/api/config/config.py

```diff
@@ -162,13 +162,9 @@ def get_forest_client_api_baseurl():
     return forest_client_api_baseurl
 
 
-def get_idim_proxy_api_baseurl(app_env: AppEnv):
+def get_idim_proxy_api_baseurl(api_instance_env: ApiInstanceEnv):
     idim_proxy_api_baseurl = get_env_var("IDIM_PROXY_BASE_URL_TEST")
```

Comment:
Nothing wrong here.
Just wondering, previously for "forest_client" the baseurl is hardcode as default value and does not need to be set in local_dev as one of the environment variable.
Do you prefer to get it for local or also hardcode the value as default?
For higher environment this will be a get().

Reply:
Hi Ian, I think about this again, and I agree we could remove it, I also did a cleanup of the terraform, almost forgot, thanks so much!